src/docfx/restore/Package.cs

```diff
@@ -21,6 +21,8 @@ internal abstract class Package
 
         public abstract DateTime? TryGetLastWriteTimeUtc(PathString path);
 
+        public abstract byte[] ReadAllBytes(PathString path);
```

Comment:
provide a default implementation like `ReadString`, then override in `LocalPackage`?